src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java

```diff
@@ -85,9 +89,8 @@ public void updateEserviceState(UpdateEserviceStateDto inputData)
 
     eServiceToUpdate.state(inputData.getNewEServiceState());
     eserviceRepository.save(eServiceToUpdate);
-
-    log.info("EserviceState of eservice " + eServiceToUpdate.eserviceId() + " with version "
-        + eServiceToUpdate.versionId() + " has been updated into " + eServiceToUpdate.state());
+    logger.logMessageEserviceStateUpdated(eServiceToUpdate.eserviceId(),
```

Comment:
Maybe it is better to pass the whole object

Reply:
Done and pushed